redaxo/src/core/lib/util/finder.php

```diff
@@ -185,7 +185,7 @@ public function sort($sort = rex_sortable_iterator::KEYS)
     }
 
     /**
-     * @return Iterator|SplFileInfo[]
+     * @return RecursiveCallbackFilterIterator|RecursiveIteratorIterator|rex_sortable_iterator
```

Comment:
```suggestion
     * @return RecursiveCallbackFilterIterator|RecursiveIteratorIterator|rex_sortable_iterator| SplFileInfo[]
```

Reply:
In deiner Suggestion ist ein Leerzeichen zu viel.

Wobei ich mich frage, ob man von außen genau mit diesen Iteratoren rechnen muss, oder nicht doch eher allgemein mit irgendeinem Iterator, der SplFileInfos liefert.